multipart/decoders.py

```diff
@@ -33,10 +35,16 @@ class Base64Decoder:
     :param underlying: the underlying object to pass writes to
     """
 
-    def __init__(self, underlying):
+    def __init__(self, underlying: IOBase):
```

Comment:
How is `underlying` a `IOBase`?

Reply:
It is a stream or file-like object with a read/write interface. From `Base64Decoder`'s docstring:
```
"""
...
        from multipart.decoders import Base64Decoder
        fd = open("notb64.txt", "wb")
        decoder = Base64Decoder(fd)
...
```
I would appreciate more feedback on this from you though.